recipes/lsd/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "lsds" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/funkelab/lsd/archive/v{{ version }}.tar.gz
+  sha256: b1afe5e78e1ef21fbc4b03b86de3577b74bb0533f056d75d339d9b6ce0f1c2ed
```

Comment:
```suggestion
{% set sha256 = "b1afe5e78e1ef21fbc4b03b86de3577b74bb0533f056d75d339d9b6ce0f1c2ed" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://github.com/funkelab/lsd/archive/v{{ version }}.tar.gz
  sha256: {{ sha256 }}
```

Reply:
This is somewhat stylistic, but it helps concentrate version update changes at the top of the file.